routing/result_interpretation.go

```diff
@@ -192,7 +192,7 @@ func (i *interpretedResult) processPaymentOutcomeFinal(
 
 		// The other hops relayed corectly, so assign those pairs a
 		// success result.
-		if n > 2 {
+		if n >= 2 {
```

Comment:
Do we need this check at all? We won't have n=0, and we return on n=1 above so at this stage n>=2 is implied?

Reply:
Fixed